### PR TITLE
DSB05 update

### PR DESCRIPTION
**Pull request by caseda**
app.json:
- remove already default values defaultConfig
- code consistency
- remove "value" in dropbox values, id = value
- change setting "Wake up 10 minutes when batteries are inserted"  to
  checkbox
- add min/max attribute number settings
- typo hint setting "On time"
- change setting "Enable motion sensor" checkbox
- removed setting "Which reports to send to association group 1" can
  stop data to homey
- change setting "Which reports to send to association group 2" dropdown
- change setting "Which reports to send to association group 3" dropdown
- typo setting "Unsolicited reports interval for association group 1"
- typo setting "Unsolicited reports interval for association group 2"
- typo setting "Unsolicited reports interval for association group 3"

driver.js:
coding consistency
removed setting "101" can stop data to homey

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -36,63 +36,33 @@
...
-                        "value": 720
-                    }, {
-                        "id": 112,
-                        "size": 4,
-                        "value": 720
-                    }, {
-                        "id": 113,
-                        "size": 4,
-                        "value": 720
+                        "value": 240
```
Comment (RobinBol) [inline app.json]: Way too often, this will very likely drain batteries, please set to default value 1800.

File: app.json [hunk 2]
```diff
@@ -123,145 +94,230 @@
                             "id": "0",
                             "label": {
                                 "en": "Temperature, humidity and luminance"
-                            },
-                            "value": 0
```
Comment (RobinBol) [inline app.json]: Why did you remove all the default values over here?

Comment (caseda) [inline app.json]: exactly as my description states:
> remove "value" in dropbox values, id = value
the id is the value

File: app.json (hunk 1 above)
Comment (caseda) [inline app.json]: then i will set it to the normal default value (720), or rather, delete the default setting 👍
